sympy/physics/control/lti.py

```diff
@@ -502,3 +420,41 @@ def is_strictly_proper(self):
     @property
     def is_biproper(self):
         return self.doit().is_biproper
+
+
+class TransferFunctionExpr(Expr):
+
+    def __new__(cls, *args, **kwargs):
+        args = map(_sympify, args)
+        obj = super().__new__(cls, *args, **kwargs)
+        return obj
+
+    def __neg__(self):
+        return Series(S.NegativeOne, self).doit()
+
+    @call_highest_priority('__radd__')
+    def __add__(self, other):
+        return Parallel(self, other, check=True)
+
+    @call_highest_priority('__add__')
```

Comment:
What exactly is this decorator for?

From what I see in its source code, don't we need to define `_op_priority` for TransferFunctions too if we want to use this?